src/themes/teams-high-contrast/components/Button/button-styles.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ComponentSlotStylesPrepared,
+  TeamsThemeStylesProps,
+} from "@fluentui/react-northstar";
+
+export const buttonStyles: ComponentSlotStylesPrepared<
+  NonNullable<Partial<TeamsThemeStylesProps["Button"]>>,
+  any
+> = {
+  root: (componentStyleParameters) => {
+    const {
+      theme: {
+        siteVariables: { colorScheme },
+      },
+      props: { primary, text },
+    } = componentStyleParameters;
+    return {
+      borderRadius: "4px",
+      borderStyle: "solid",
+      "&:hover": {
+        borderColor: colorScheme.default.borderHover,
+        backgroundColor: colorScheme.default.backgroundHover,
+        color: colorScheme.default.foregroundHover + " !important",
```

Comment:
I think this may be a bug, either in Fluent UI or Fela, since these styles are more local and should override the styles they're being merged with, and that's true for the base styles but not the styles applied for these pseudo-classes.

I may need help avoiding `!important`… @levithomason, would you know a workaround, perhaps?